modules/auth_saml/config/locales/en.yml

```diff
@@ -1,5 +1,5 @@
 en:
-  activemodel:
+  activerecord:
```

Comment:
Great catch!

Now the `I18n.t` calls with `activemodel.attributes.saml/provider` need to be updated too (searching for it revealed 19 occurences). Either adapted to use `human_attribute_name` or simply update the key.

Reply:
Indeed, same with OIDC actually